flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/co/CoBroadcastWithKeyedOperator.java

```diff
@@ -324,6 +324,11 @@ public TimeDomain timeDomain() {
 			return timeDomain;
 		}
 
+		@Override
+		public KS getCurrentKey() {
```

Comment:
It needs at least some most basic test coverage

Reply:
added